extensions/opc/src/opc.cpp

```diff
@@ -302,7 +303,7 @@ void Client::traverse(UA_NodeId nodeId, std::function<nodeFoundCallBackFunc> cb,
 
   UA_BrowseResponse bResp = UA_Client_Service_browse(client_, bReq);
 
-  utils::ScopeGuard guard([&bResp]() {
+  gsl::finally guard([&bResp]() {
```

Comment:
`gsl::finally` is a function, not a type. 
```suggestion
  const auto guard = gsl::finally([&bResp] {
```

Reply:
PR updated with the correct signature.